### ISSUE TITLE
User page

### ISSUE THREAD

**Issue by sprucely** [ucdavis/hippo#233]
A landing page for users to display:

- What resources they have
- What groups they are in
- How to request a different sponsor

**Comment by sprucely** [comment]
@acgetchell What resources were you thinking when wrote the above? (It was saying you wrote it until I converted it to an issue.) Looking at user account yaml, I see things like
```
    shell: /bin/bash
    ensure: present
    storage:
      zfs: false
      autofs:
        nas: nas-4-1-ib
        path: /nas-4-1/bennettgrp2/
    slurm:
      account:
      - bennettgrp
```
Then under group yaml, I see additional storage and slurm details.

**Comment by sprucely** [comment]
Addressed by #243
